build.gradle

```diff
@@ -14,13 +14,13 @@ buildscript {
         android_components: '15.0.0',
         android_gradle_plugin: '3.5.3',
         android_maven_publish_plugin: '3.6.2',
-        coroutines: '1.3.0',
+        coroutines: '1.3.5',
         dokka: '0.9.17',
         jna: '5.2.0',
         junit: '4.12',
         mockito: '2.24.5',
         mockwebserver: '3.10.0',
-        kotlin: '1.3.50',
+        kotlin: '1.3.71',
         robolectric: '4.2.1', // This is different than a-c, but we're fine, it's only tests.
         rust_android_plugin: '0.8.3',
```

Comment:
Any chance you could also double check if A-C is using any newer version for `androidx_*` stuff below? They are, at least, for `androidx_core`

Reply:
Yep, let me go through and update everything I can find...   commits incoming.